--- gsmlib-1.10/gsmlib/gsm_map_key.h.orig.1	2005-09-12 13:33:35.000000000 +0100
+++ gsmlib-1.10/gsmlib/gsm_map_key.h	2005-09-12 13:33:55.000000000 +0100
@@ -75,7 +75,7 @@
   // MapKey members
   
   template <class SortedStore>
-    bool gsmlib::operator<(const MapKey<SortedStore> &x,
+    bool operator<(const MapKey<SortedStore> &x,
                            const MapKey<SortedStore> &y)
     {
       assert(&x._myStore == &y._myStore);
@@ -100,7 +100,7 @@
     }
 
   template <class SortedStore>
-    bool gsmlib::operator==(const MapKey<SortedStore> &x,
+    bool operator==(const MapKey<SortedStore> &x,
                             const MapKey<SortedStore> &y)
     {
       assert(&x._myStore == &y._myStore);
